app/jobs/remove_old_throttles_job.rb

```diff
@@ -0,0 +1,43 @@
+class RemoveOldThrottlesJob < ApplicationJob
+  queue_as :low
+
+  WINDOW = 30.days.freeze
+
+  include GoodJob::ActiveJobExtensions::Concurrency
+
+  good_job_control_concurrency_with(
+    enqueue_limit: 1,
+    perform_limit: 1,
+    key: -> do
+      rounded = ApplicationJob.round_time(time: arguments.first, interval: 1.hour)
+      "remove-old-throttles-#{rounded.to_i}"
+    end,
+  )
+
+  discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
+
+  def perform(now, limit: 1000, total_limit: 100_000)
```

Comment:
I added `total_limit` because there are throttle rows in prod, so when those expire, I wanted to be certain this job would not run too long